CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Add the reserved status code AudioDisconnectAudio.
-
+- Add support in `MessagingSession` to allow websocket connection for messaging service to enable Prefetch feature.
```

Comment:
If there is a guide for the `Prefetch` feature would be good to add a link here.

nit: Add an extra line after this.

Reply:
The prefetch feature is not officially launched yet, so we cannot add the link yet lol.

Reply:
The prefetch feature is not officially launched yet, so cannot link any doc here for now lol.

Reply:
I think you can just update the README to mention that you can set prefetch option to enable this feature.